server/routes/induction/update/qualificationsListUpdateController.ts

```diff
@@ -17,23 +17,9 @@ export default class QualificationsListUpdateController extends QualificationsLi
   getBackLinkUrl(req: Request): string {
     const { prisonNumber } = req.params
     const { pageFlowHistory } = req.session
-    if (pageFlowHistory) {
```

Comment:
This TODO was a hangover from the previous (very complicated) long and short journeys.
The replacement code is what we do on all the other pages in the new question set, and didnt break any tests
We know that we do test back link locations pretty thoroughly at the moment, so I am confident that the existing tests are correct/sufficient; and that the previous implementation worked but was overkill